packages/components/_private/date-panel/src/types.ts

```diff
@@ -5,19 +5,24 @@
  * found in the LICENSE file at https://github.com/IDuxFE/idux/blob/main/LICENSE
  */
 
-import type { ExtractInnerPropTypes, ExtractPublicPropTypes } from '@idux/cdk/utils'
-import type { DefineComponent, HTMLAttributes } from 'vue'
-
-import { IxPropTypes } from '@idux/cdk/utils'
+import type { ExtractInnerPropTypes, ExtractPublicPropTypes, MaybeArray } from '@idux/cdk/utils'
+import type { DefineComponent, HTMLAttributes, PropType } from 'vue'
 
 export const datePanelProps = {
-  disabledDate: IxPropTypes.func<(date: Date) => boolean>(),
-  type: IxPropTypes.oneOf<DatePanelType>(['date', 'week', 'month', 'quarter', 'year']).def('date'),
-  value: IxPropTypes.object<Date>(),
-  visible: IxPropTypes.bool,
-
-  onCellClick: IxPropTypes.emit<(date: Date) => void>(),
-  onCellMouseenter: IxPropTypes.emit<(date: Date) => void>(),
+  cellTooltip: Function as PropType<(cell: { value: Date; disabled: boolean }) => string | void>,
+  disabledDate: Function as PropType<(date: Date) => boolean>,
+  type: {
+    type: String as PropType<DatePanelType>,
+    default: 'date',
+  },
+  value: [Date, Array] as PropType<Date | (Date | undefined)[]>,
+  activeDate: Date,
+  visible: Boolean,
+  isSelecting: Boolean,
```

Comment:
```suggestion
  visible: { type: Boolean,default: undefined },
  isSelecting:  { type: Boolean,default: undefined },
```
对于 String, Number, Boolean 这种基本类型，手动给默认值吧，否则 vue 会给默认值，有些时候会有意料之外的惊喜。。。